tests/integration/providers/ldap/delta_inbound_sync_tests.py

```diff
@@ -583,7 +581,32 @@ def test_create_fake_user(ldap_connection, user):
     time.sleep(1)
     email = "%s@clouddev.corporate.t-mobile.com" % user["common_name"]
     result = is_user_in_db(email)
+    syncflag_fetched = get_syncmatch_count(user["common_name"])
     assert result is True
+    assert syncflag_fetched is True
+
+
+def get_syncmatch_count(username_provided):
```

Comment:
it may be nitpicky but the function name implies that it's returning an int count of something, something like 'is_user_outbound' implies a boolean

Reply:
sure, will change the name to  'is_user_inbound()'